gateway_rejection_reason.go

```diff
@@ -0,0 +1,14 @@
+package braintree
+
+type GatewayRejectionReason string
+
+const (
+	GatewayRejectionReasonApplicationIncomplete GatewayRejectionReason = "application_incomplete"
+	GatewayRejectionReasonAVS                                          = "avs"
+	GatewayRejectionReasonAVSAndCVV                                    = "avs_and_cvv"
+	GatewayRejectionReasonCVV                                          = "cvv"
+	GatewayRejectionReasonDuplicate                                    = "duplicate"
+	GatewayRejectionReasonFraud                                        = "fraud"
+	GatewayRejectionReasonThreeDSecure                                 = "three_d_secure"
+	GatewayRejectionReasonUnrecognized                                 = "unrecognized"
```

Comment:
The only identifier that is typed in this list is the first one, the others are untyped consts. e.g. https://play.golang.org/p/sK63YlkNXk